### DISCUSSION TITLE
¡Se me cerro el pull request!

### DISCUSSION THREAD

**Discussion by luciano-repetti** [INGCOM-UNRN-P1/cursada-2023#16] [Preguntas y respuestas] [answered]
Buenas. Se me cerro el pull request y no me deja re abrirlo o no se como.
¿Como podría reabrir el PR o empezar de 0 el repo?

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/67850074/694b7963-0c86-4eba-b6c2-f8eca07acac4)

PD: No cerre a posta el pull request, se me cerro por un ```git push origin main --force``` por estar probando cositas.

**Comment by martinvilu** [comment, marked as answer]
Ahí vi que lograse volver a abrirlo. 👍
